app/uk/gov/hmrc/apiplatform/modules/submissions/domain/services/SubmissionDataExtracter.scala

```diff
@@ -57,12 +57,24 @@ object SubmissionDataExtracter extends ApplicationLogger {
     getTextQuestionOfInterest(submission, submission.questionIdsOfInterest.organisationUrlId)
   }
 
-  def getResponsibleIndividualName(submission: Submission): Option[ResponsibleIndividual.Name] = {
-    getTextQuestionOfInterest(submission, submission.questionIdsOfInterest.responsibleIndividualNameId).map(ResponsibleIndividual.Name)
+  def getResponsibleIndividualName(submission: Submission, requestedByName: String): Option[ResponsibleIndividual.Name] = {
+    val responsibleIndividualIsRequesterId = submission.questionIdsOfInterest.responsibleIndividualIsRequesterId
+    val yesOrNoResponsibleIndividualIsRequester = getSingleChoiceQuestionOfInterest(submission, responsibleIndividualIsRequesterId)
+println("yesOrNoResponsibleIndividualIsRequester " + yesOrNoResponsibleIndividualIsRequester)
```

Comment:
Did you leave the println in deliberately?

Reply:
haha - no, will fix this thanks